source/docs/3/river_styling_expressions.rst

```diff
@@ -0,0 +1,109 @@
+Styling a river network with Expressions (QGIS3)
+===============================
+*Contributed by:* `Steven Kim <https://geographyclub.github.io/>`_
+
+The HydroRIVERS dataset provides seamless global coverage of consistently sized river reaches, supported by geometric information that allows for basic analysis of river network topology such as stream connectivity and distances. The data is a subset of the comprehensive HydroATLAS package from WWF.
+
+Overview of the task
+--------------------
+
+We will use expressions to visualize South American rivers in a popular style, with line widths representing upland area and colors representing basin id from HydroRIVERS.
+
+Get the data
+------------
+
+1. `HydroRIVERS <https://www.hydrosheds.org/products/hydrorivers>`_ has links for global and regional extracts of HydroRIVERS in shapefile and geodatabase formats. Look for the links at the bottom of the page. 
+
+  .. image:: /static/3/river_styling_expressions/images/data1.png
+    :align: center
+	
+2. Click on the download link for the ``South America`` shapefile. You will see the ``HydroRIVERS_v10_sa_shp.zip`` file downloaded which is used for this tutorial.
+
+  .. image:: /static/3/river_styling_expressions/images/data1.png
+    :align: center
+
+For convenience, you may directly download a copy of the above layer from below:
+`ne_50m_admin_0_countries.zip <https://www.qgistutorials.com/downloads/HydroRIVERS_v10_sa.shp.zip>`_
+
+Procedure
+---------
+
+1. Locate the ``HydroRIVERS_v10_sa_shp.zip`` file in the QGIS Browser and expand it. Select the ``HydroRIVERS_v10_sa_shp/HydroRIVERS_v10_sa.shp`` file and drag it to the canvas.
+
+  .. image:: /static/3/river_styling_expressions/images/1.png
+    :align: center
+
+2. A new layer ``HydroRIVERS_v10_sa`` will now be loaded in QGIS and you should see a map of rivers in South America. Click on :guilabel:`Open Layer Styling Panel`.
+
+  .. image:: /static/3/river_styling_expressions/images/2.png
+    :align: center
+
+3. Switch from :guilabel:`Single Symbol` to :guilabel:`Rule-based`.
+
+  .. image:: /static/3/river_styling_expressions/images/3.png
+    :align: center
+	
+4. Double-click on the default rule to enter the :guilabel:`Edit Rule` dialog.
+
+  .. image:: /static/3/river_styling_expressions/images/4.png
+    :align: center
+
+5. To filter using an expression, click the Filter :guilabel:`ε`. 
+
+  .. image:: /static/3/river_styling_expressions/images/5.png
+    :align: center
+
+.. note:: 
+
+You have entered the :guilabel:`Expression Builder`. If this is your first time working with expressions, see the official docs at `QGIS User Guide <https://docs.qgis.org/3.34/en/docs/user_manual/expressions/expression.html>`_ 
```

Comment:
use the 'testing' doc reference so we don't have to update it. https://docs.qgis.org/testing/en/docs/user_manual/expressions/expression.html